ros2_batch_job/__main__.py

```diff
@@ -85,8 +84,9 @@ def get_args(sysargv=None):
         help='create an archive of the install space')
     parser.add_argument(
         '--repo-file-url',
-        default=DEFAULT_REPOS_URL,
-        help="url of the ros2.repos file to fetch and use for the basis of the batch job")
+        help="url of the ros2.repos file to fetch and use for the basis of the batch job",
+        required=True)
```

Comment:
By my reading of the argparse docs this should be all I need to make this parameter required.